cypress/integration/page.spec.js

```diff
@@ -1026,7 +1104,7 @@
       page.spyOnCommentsPost();
 
       newCommentForm.emailInput().type('some@email.com');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
       newCommentForm.submitButton().click();
 
       cy.get('@commentsPost').should('not.be.called');
```

Comment:
The method `cy.get('@commentsPost').should('not.be.called')` should be `cy.get('@commentsPost').should('not.have.been.called')` to correctly assert that the spy was not called.